pkg/apis/garden/v1beta1/types.go

```diff
@@ -107,15 +107,26 @@ type AWSConstraints struct {
 	Zones []Zone `json:"zones"`
 }
 
-// MachineImage defines the name and the version of the machine image in any environment.
+// MachineImage defines the name and multiple versions of the machine image in any environment.
 type MachineImage struct {
 	// Name is the name of the image.
 	Name string `json:"name"`
+	// DEPRECATED: This field will be removed in a future version.
+	Version string `json:"version"`
```

Comment:
```suggestion
	// +optional
	Version string `json:"version,omitempty"`
```

Otherwise, I cannot PUT a `CloudProfile` looking like this:

```yaml
      machineImages:
      - name: coreos
        versions:
        - version: 1967.5.0
        - version: 2023.4.0
```

without specifying `version: ""`. It should be fine making a field optional that was required before.